appcli/models/cli_context.py

```diff
@@ -31,6 +31,24 @@
 IGNORE_INFIX = ".appcli"
 """ An infix extension to ignore when copying schema files. """
 
+HELM_DIRECTORY = Path("cli/helm")
+""" Directory for helm config data. """
+
+HELM_VALUES = HELM_DIRECTORY / "values"
+""" The directory containing all main `values.yaml` files.
+All files in this directory are applied with:
+    --values <file>
+"""
+
+HELM_VALUES_FILES = HELM_DIRECTORY / "values-files"
+""" The directory containing all key-specific files.
+All parent directories for a file are used as nested keys.
+The [0] filename component (split on `.`) is used for the key.
+
+All files in this directory are applied with:
+    --set-file path.to.foo=/path/to/foo.bar.yml
+"""
```

Comment:
This is hard to follow. It's worth putting in the actual example similar to the README which is really clear.